mmdet/datasets/voc.py

```diff
@@ -22,7 +22,7 @@ class VOCDataset(XMLDataset):
     }
 
     def __init__(self, **kwargs):
-        super().__init__(**kwargs)
+        super().__init__(minus_one=True, **kwargs)
```

Comment:
Seems this is a hard code. How about set a default value?

Reply:
@chhluo Can you take a look at it? And give some suggestions.